dsl_parser/utils.py

```diff
@@ -348,3 +348,7 @@ def is_blueprint_import(import_url):
 
 def remove_blueprint_import_prefix(import_url):
     return import_url.replace(constants.BLUEPRINT_IMPORT, '')
+
+
+def find_partial_matches_in_list(value, items):
```

Comment:
Please name it to be more user-friendly rather than 'value' and write a little description of it's usage